raft/src/main/java/org/apache/kafka/raft/ValidOffsetAndEpoch.java

```diff
@@ -25,7 +27,7 @@
         this.offsetAndEpoch = offsetAndEpoch;
     }
 
-    public Type type() {
+    public Type getType() {
```

Comment:
By the way we can also just change the name of the type and field to something like `public Kind kind()`